### ISSUE TITLE
Autorank 3.7.3 does not promote - 3.7.2 generates a lot of console errors

### ISSUE THREAD

**Issue by Soz3r** [Staartvin/Autorank-2#310]
Here is the section of the advancedconfig that uses worldguard region: http://pastebin.com/36hKgPie
Console says "Autorank: Config files have been correctly setup!"

I made the region quite sizable to hopefully improve chances of it working.  The worldguard region matches the case of the region specified in the Autorank config. Using the command "/ar hooks" shows worldguard is an available hook.

What I've done to try to fix it:
Tried different region
Tried /ar check while meeting all requirements
Purged all player data, started fresh
Reinstalled AutoRank - (kept configs)

I'm hoping you see an error in the pastebin, or this is a known issue - it would explain a lot.

**Comment by Staartvin** [comment]
Hmm, what version of Autorank and Worldguard are you using?

**Comment by Soz3r** [comment]
Autorank 3.7.3 
Worldguard 6.1
MC: 1.9.2

**Comment by Staartvin** [comment]
Hmm, this is odd. I've tested it and it seems to work for me. Did you make sure it is typed correctly? It is case-sensitive.

**Comment by Soz3r** [comment]
I was testing last night and I need to isolate this issue.  Its not just world guard anymore.  I reset all player data again, added myself to the default group and joined the server fresh.  After the first promotion of 15 minutes it still didn't promote me. 

Is there any typical mistake people make that would stop Autorank from promoting?  Is it something with the config?  

I remember getting errors in my config when I used certain types of fireworks.  I will try taking out firework results tonight and see if that, for some reason, effects it.

**Comment by Staartvin** [comment]
Could you show me your whole advanced config and server log? Most problems will be shown there :=

**Comment by Soz3r** [comment]
Server log: http://pastebin.com/AFp7Wr3C
Adv config: http://pastebin.com/y19mPwh9

Also, pasting that in reminded me, I want to know if there is another source for the reference links at the bottom of the default adv config for effects and firework types.  Those jd.bukkit.org links have shown 404 errors since I first tried accessing them (about a week ago).

**Comment by Staartvin** [comment]
It seems like your AdvancedConfig is correct and the server log doesn't show any errors. Could you try making a 'TestRegion' and also set the worldguard region requirement to 'TestRegion'? That should definitely work (as I've tried that myself just now).

For the links, see this updated AdvancedConfig file: https://github.com/Armarr/Autorank-2/blob/master/AdvancedConfig.yml

**Comment by Soz3r** [comment]
![2016-04-14_18 08 19](https://cloud.githubusercontent.com/assets/4951769/14545308/0c92124e-026c-11e6-8dd0-79bb9f1a39d1.png)

Tried it with a different region, also cleared all plugin data prior to it.  Also I thought the problem coincided with when I added all the firework results.  So I removed those from the config as well with still no luck :(.   I think I updated the config and the plugin to 3.7.3 - I cant find the old 3.7.2, is there a repository I can try an old version?
